tests/test_e2e_40_sdntrace.py

```diff
@@ -278,9 +278,8 @@ def test_020_run_sdntrace_fail_missing_flow(self):
         api_url = KYTOS_API + '/amlight/sdntrace_cp/v1/trace'
         response = requests.put(api_url, json=payload_1)
         data = response.json()
-        # only 4 steps are expected: starting, 1->2, 2->3, 3->4, 4->5(incomplete)
-        assert len(data["result"]) == 5, str(data)
-        assert data["result"][-1]['type'] == "incomplete"
+        # only 4 steps are expected: starting, 1->2, 2->3, 3->4
+        assert len(data["result"]) == 4, str(data)
```

Comment:
Hi Gretel,

I'm having some doubts here. At this point here, before deleting the flow, there will be flows from switch `00:..:01` and `00:..:0a`, right? In this case, we will have ten steps, right? (starting, 1->2, 2->3, 3->4, 4->5, 5->6, 6->7, 7->8, 8->9, 9->a). Since we removed the flows from switch 5, we will have flows from 1->2, 2->3, 3->4 plus the starting step. Wouldn't that count 5 on the `len(data["result"])` assertion above?

Reply:
@italovalcy, Thank you for your comment. Please, note that `starting` is for the first step with sw1, the second step is for sw2, step 3 is for sw3 and step 4 is for sw4. sw5 has been deleted as you mentioned:
 
```
{'result': [{'dpid': '00:00:00:00:00:00:00:01', 'port': 1, 'time': '2023-08-09 18:48:23.743565', 
                     'type': 'starting', 'vlan': 400}, {'dpid': '00:00:00:00:00:00:00:02', 'port': 2, 
                    'time': '2023-08-09 18:48:23.743614', 'type': 'intermediary', 'vlan': 1}, 
                    {'dpid': '00:00:00:00:00:00:00:03', 'port': 2, 'time': '2023-08-09 18:48:23.743631',
                      'type': 'intermediary', 'vlan': 1}, {'dpid': '00:00:00:00:00:00:00:04', 'port': 2, 
                    'time': '2023-08-09 18:48:23.743644', 'type': 'intermediary', 'vlan': 1, 
                    'out': {'port': 3, 'vlan': 1}}]}
```